src/css/styles.scss

```diff
@@ -274,3 +340,15 @@ main {
   background-image: url(https://image.flaticon.com/icons/svg/148/148841.svg);
   background-repeat: no-repeat;
 }
+
+@media (min-width: 800px) {
```

Comment:
Thanks so much for adding these media queries in! From the screenshot, they look awesome.

Reply:
I agree your spacing looks great!